### DISCUSSION TITLE
Deploy django app

### DISCUSSION THREAD

**Discussion by TheusMarqs** [vercel/community#6560] [Help] [answered]
### Summary

I have a django project with python 3.9.13 but when I try to deploy it, Vercel shows this error:

Unable to find any supported Python versions.

### Steps to Reproduce

my vercel.json has the right python version, pipfile, pipfile.lock too and I don't know what else I can do, can someone help me?

**Comment by JINYUHOON** [comment, marked as answer]
When I had the same problem before, I had experience solving it by changing the configuration structure of vercel.json.
However, it runs in flask, not django.
Additionally, requirements.txt is included in the root folder.

vercel.json
```
{
    "version" : 2,
    "builds": [
        {
            "src": "application.py",
            "use": "@vercel/python",
            "config": { "maxLambdaSize": "15mb", "runtime": "python3.9" }
        }
    ],
    "routes": [
        {
            "src": "/(.*)",
            "dest": "application.py"
        }
    ]
  }
```

**Reply by TheusMarqs** [reply]
It worked!! thank u so much bro!
I didn't have this line: "version" : 2
